### ISSUE TITLE
Missing Mysql Options tab in Settings 

### ISSUE THREAD

**Issue by jimiero** [watchfulli/XCloner-Wordpress#277]
With our latest version we are missing the "Mysql Options" tab in XCloner settings

Reproduced on our demo account "006 - Empire Estates"

![Screenshot 2023-03-02 at 1 29 53 PM](https://user-images.githubusercontent.com/3626762/222416533-0be424ac-562a-4479-a0cd-aec95c6e80ff.png)

**Comment by Ars92** [comment]
@jimiero We removed that tab on version 4.4.5 as it wasn't used anymore by XCloner.
Why do you need that tab?

**Comment by jimiero** [comment]
Oh, a customer says he is seeing some mysql databases unrelated with the one of his website, for example "information_schema" from what i know that's used by mysql itself

**Comment by Ars92** [comment]
@jimiero 
I agree that's an error an we should only show the database WordPress is using but having that tab back won't solve the issue.
Can you please try to replicate and update the title and the description of this issue?

**Comment by jimiero** [comment]
@Ars92 i confirm we display non wanted databases also can be seen on the same demo website.

**Comment by Ars92** [comment]
@jimiero 
>I agree that's an error an **we should only show the database WordPress is using but having that tab back won't solve the issue**.
Can you please try to replicate **and update the title and the description of this issue**?

Since you are able to replicate, you should update this issue title and description.
We don't want to show DB that are not the one used by WordPress. We don't want that tab back.

**Comment by jimiero** [comment]
updated
